assignments/array-methods.js

```diff
@@ -55,29 +55,86 @@ const runners = [{"id":1,"first_name":"Charmain","last_name":"Seiler","email":"c
 
 // ==== Challenge 1: Use .forEach() ====
 // The event director needs both the first and last names of each runner for their running bibs.  Combine both the first and last names into a new array called fullName. 
-let fullName = [];
+const fullName = [];
+
+runners.forEach(element => {
```

Comment:
Try to use a more descriptive name than element when using array methods --> runner could be a good name here